action.yml

```
@@ -13,6 +13,9 @@ inputs:
   github-token:
     description: "GitaHub Token"
     required: true
+  mobb-project-name:
+    description: "Mobb Project Name"
+    required: false
```

@antonychiu2 please take into account that in case the user does not enter project name it will be null
therefore the command 
`--mobb-project-name ${{ inputs.mobb-project-name }})`
will be 
`--mobb-project-name `
therefore it wil fail 

we can solve it by giving it our project name default value

```suggestion
  mobb-project-name:
    description: "Mobb Project Name"
    required: false
    default: 'My first project'
```

---

hey @antonychiu2  I've checked and our code set a default value even if we provide an empty string.
you can leave the code as is